modules/database/database-sql-hana/src/main/java/org/eclipse/dirigible/database/sql/dialects/hana/HanaDropSequenceBuilder.java

```diff
@@ -67,7 +67,11 @@ public String generate() {
      * @param sql the sql
      */
     protected void generateRestrict(StringBuilder sql) {
-        String sequenceName = (isCaseSensitive()) ? encapsulate(this.getSequence(), true) : this.getSequence();
```

Comment:
This whole method seems odd. Is it actually doing something on this line?

Reply:
fixed